com.unity.cinemachine/Runtime/Behaviours/CinemachineConfiner.cs

```diff
@@ -271,22 +290,21 @@ private Vector3 ConfinePoint(Vector3 camPos)
 #endif
         }
 
-        // Camera must be orthographic
-        private Vector3 ConfineScreenEdges(CinemachineVirtualCameraBase vcam, ref CameraState state)
+        Vector3 ConfineOrthoCameraToScreenEdges(CinemachineVirtualCameraBase vcam, ref CameraState state)
```

Comment:
input parameter `vcam` can be removed - just like in https://github.com/Unity-Technologies/com.unity.cinemachine/pull/507 - [CMCL-1070](https://jira.unity3d.com/browse/CMCL-1070).

Reply:
Resolved by 4ed1486c700cd99ebb26a5e986a9f6626f977814